teku/src/main/java/tech/pegasys/teku/cli/options/DepositOptions.java

```diff
@@ -26,6 +28,15 @@
       arity = "1")
   private String eth1Endpoint = null;
 
+  @Option(
+      names = {"--eth1-endpoints"},
+      paramLabel = "<NETWORK>",
+      description = "URLs for Eth1 nodes.",
+      split = ",",
+      arity = "1..*",
+      hidden = true)
+  private List<String> eth1Endpoints = null;
```

Comment:
We can probably just set the default here to `new ArrayList()` and I think it will work out fine so we don't need null checks later.